ext/pdo_mysql/tests/pdomysql_001.phpt

```diff
@@ -0,0 +1,47 @@
+--TEST--
+PDO_mysql subclass basic
+--EXTENSIONS--
+pdo_mysql
+--SKIPIF--
+<?php
+require_once __DIR__ . '/inc/mysql_pdo_test.inc';
+MySQLPDOTest::skip();
+?>
+--FILE--
+<?php
+
+require_once __DIR__ . '/inc/mysql_pdo_test.inc';
+
+$db = MySQLPDOTest::factory(PdoMysql::class);
+
+$db->query('CREATE TABLE IF NOT EXISTS pdomysql_001 (id INT, name TEXT)');
```

Comment:
```suggestion
$db->query('CREATE TABLE pdomysql_001 (id INT, name TEXT)');
```
The CLEAN section should ensure this always works

Reply:
Nice catch, I totally missed this one when I reworked Danack's PR.